commitizen/git.py

```diff
@@ -44,13 +44,20 @@ def __eq__(self, other) -> bool:
 
 class GitCommit(GitObject):
     def __init__(
-        self, rev, title, body: str = "", author: str = "", author_email: str = ""
+        self,
+        rev,
+        title,
+        body: str = "",
+        author: str = "",
+        author_email: str = "",
+        parents: list[str] = [],
     ):
         self.rev = rev.strip()
         self.title = title.strip()
         self.body = body.strip()
         self.author = author.strip()
         self.author_email = author_email.strip()
+        self.parents = parents
```

Comment:
Then handle the default here:
```suggestion
        self.parents = parents or []
```

Reply:
Done.